### PR TITLE
Ui mailer updates

### PR DESCRIPTION
**Pull request by cleargif**

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -167,37 +167,121 @@ en:
...
+      firefox_message: |
+        <strong style="font-weight: 700">Internet Explorer 6 and 7 users:</strong>
+        <br/>
+        We recommend the <strong style="font-weight: 700">Firefox</strong> web browser for a more enhanced experience.
+        <br/><br/>
+        <strong style="font-weight: 700">Copy and paste</strong> the link below to use in <strong style="font-weight: 700">Firefox</strong>
   reminder_mailer:
     information_request:
       subject: "Request to update your People Finder profile"
+      firefox_message: |
```
Comment (jsugarman) [inline config/locales/en.yml]: Can these repetitive translations be DRY'd out - using anchors?

**Comment by jsugarman** [comment]
Aside form needed a rebase and my inline comment about repetitive translations then if you are happy I would 👍

**Comment by cleargif** [comment]
@jsugarman done the DRY'ing 

not doing the subjects right now - think we will review them soon anyway

File: app/views/shared/_email_flash_message.haml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+- if @firefox_browser_warning.present?
+  %tr
+    %td{align: "left", valign: "top", width: "560", style: "line-height: 1.5"}
+      %p{style: "border: 4px solid red; padding: 10px; font-size: 16px;"}
+        = @firefox_browser_warning.html_safe
+        %br/
+        %br/
+        debug
```
Comment (jsugarman) [inline app/views/shared/_email_flash_message.haml]: left over?

File: app/views/shared/_email_flash_message.haml [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+        = link_url.gsub('http://','h&#65279;<span>t&#65279;</span>&#65279;tp<span>:&#65279;/</span>&#65279;/').html_safe
```
Comment (jsugarman) [inline app/views/shared/_email_flash_message.haml]: is it worth having this in here if we know it does not work?

File: app/mailers/user_update_mailer.rb [hunk 4]
```diff
@@ -8,6 +8,7 @@ def new_profile_email(person, by_email = nil)
     @person = person
     @by_email = by_email
     @profile_url = profile_url(person)
+    @firefox_browser_warning = t('.firefox_message', default: '')
```
Comment (jsugarman) [inline app/mailers/user_update_mailer.rb]: you should be able to DRY these out using
```
before_action :set_firefox_warning,
    :new_profile_email,
    :updated_profile_email,
    :deleted_profile_email
```

and then at bottom of page in private section

```
private

  def set_firefox_warning
    @firefox_browser_warning = t('.firefox_message', default: '')
  end
```

Comment (jsugarman) [inline app/mailers/user_update_mailer.rb]: but test this works using mailer previews first as i have not checked this works

File: app/mailers/reminder_mailer.rb [hunk 5]
```diff
@@ -1,6 +1,9 @@
 class ReminderMailer < ActionMailer::Base
   include FeatureHelper
   layout 'email'
+
+  before_action :set_firefox_warning
+
   add_template_helper PeopleHelper
```
Comment (cleargif) [inline app/mailers/reminder_mailer.rb]: @jsugarman can you check the others.. seems to be working locally

**Comment by jsugarman** [comment]
looks good to me 👍

**Pull request merged**